network/load-balancer/quickstart.mdx

```diff
@@ -6,78 +6,86 @@ content:
-      <Lightbox src="scaleway-lb-az.webp" />
-    - Select a Load Balancer **type**.
-    - Select an **IP address** for your Load Balancer. You can either allocate a new IP, or select one of your existing IPs.
-    - Configure the frontend and backend.
-      - **Configure frontend**. Enter a name an a port for your [frontend](/network/load-balancer/concepts/#frontends). The frontend will listen on this port, and forward received requests to the backend(s). Note that you cannot add an SSL certificate at this stage, but you can [add one after creation of the Load Balancer](/network/load-balancer/how-to/setup-ssl-offloading/).
-      - **Configure backend**. Enter the following information for your [backend](/network/load-balancer/concepts/#backends):
-        - Backend name
-        - [Protocol](/network/load-balancer/concepts/#protocol): select TCP or HTTP depending on your required usage. The protocol is used for data transmission between the frontend/s and backend/s.
-        - Port: enter the port your backend will listen on.
-        - Proxy: select a [proxy protocol](/network/load-balancer/concepts/#proxy-protocol) option from the list. Depending on the type of health check you choose, you are prompted for additional information, for example HTTP healthchecks require a method (GET, PUT or POST, a URI and a response code).
-        - Server IPs: enter the IP addresses of each of your backend servers.
-    - Configure **advanced settings** if required:
-      - **S3 failover**: activate the feature and enter a Scaleway Object Storage [Bucket Website](/storage/object/concepts/#bucket-website) URL. If none of your backend servers are available, users will be redirected to this page. 
+      - `WAW`: Warsaw, Poland.
+
+5. Select a Load Balancer **type**, according to your bandwidth needs.
+    <Lightbox src="scaleway-select-load-balancer.webp" alt="" />
+
+6. Select an **IP address** for your Load Balancer. You can either allocate a new IP, or select one of your existing IPs.
+
+7. Configure your Load Balancer's **[frontend](/network/load-balancer/concepts/#frontends)** by entering a **name** and a **port**. The frontend will listen on this port, and forward the requests it receives from clients to the backend(s). Note that you cannot add an SSL certificate at this stage, but you can [add one after the creation of the Load Balancer](/network/load-balancer/how-to/add-certificate/). You can also add [additional frontends](#how-to-add-frontends) after creation.
+    <Lightbox src="scaleway-configure-frontend.webp" alt="" />
 
+8. Configure your Load Balancer's **[backend](/network/load-balancer/concepts/#backends)** by entering a **name**, and the **protocol** and **port** it should use to connect to the backend servers. Enter also a **proxy**, **health check** method and the **server IPs** of the backend servers it should direct traffic to. You can [add additional backends](#how-to-add-backends) after creating your Load Balancer, if required.
```

Comment:
The mention to proxy is not really clear. It would be better to say something like "select a proxy protocol option"